requirements.txt

```diff
@@ -1,11 +1,10 @@
 fastapi
 gradio
-ibm-generative-ai
-ibm_watson_machine_learning
 kubernetes
 langchain
 llama_index
 torch
 transformers
 uvicorn
-redis
\ No newline at end of file
+redis
+-r src/llms/requirements.txt
```

Comment:
is this the preferred approach vs just breaking up the deps into sections/groups w/ comments?

Reply:
This maintains a dependency list of libraries only used by the class with the class for easy removal if a specific backend library is not compatible with future Python versions or is not included with the final build. Once we introduce a dependency management tool, these should move to it as well.